src/cmd/CMakeLists.txt

```diff
@@ -7,7 +7,13 @@ set(cmd_script_configured "${cmd_script_generated}.configured")
 
 # Set the library_location variable to the relative path to the library file
 # within the install directory structure.
-set(library_location "../../../${CMAKE_INSTALL_LIBDIR}/$<TARGET_FILE_NAME:${ign_lib_target}>")
+if(WIN32)
+  set(plugin_location ${CMAKE_INSTALL_BINDIR})
+else()
+  set(plugin_location ${CMAKE_INSTALL_LIBDIR})
+endif()
+
+set(library_location "../../../${plugin_location}/$<TARGET_FILE_NAME:${ign_lib_target}>")
 
```

Comment:
Windows CI is failing on this file:

```
[12.204s] CMake Error in src/cmd/CMakeLists.txt:
[12.204s]   Evaluation file to be written multiple times with different content.  This
[12.204s]   is generally caused by the content evaluating the configuration type,
[12.204s]   language, or location of object files:
[12.204s] 
[12.204s]    C:/Jenkins/workspace/ign_gazebo-pr-win/ws/build/ignition-gazebo6/test/lib/ruby/ignition/cmdgazebo6.rb
```

Reply:
Thanks. For reference, I was developing using Ninja cmake generator, while this error appears with `Visual Studio 16 2019`.

Reply:
This turned out to be quite a can of worms. Basically, all the logic that is generating .yaml files and ruby scripts for testing was written strictly assuming the use of a single-config CMake generator. So everything was working fine when using make, single-config Ninja or NMake Makefiles, but it was going to fail as soon as Visual Studio, XCode or multiple config Ninja was used.

Initially, I tried to make the CMake code support also multiple config generator, but that was going to make that code quite difficult to mantain. For this reason, I switched the PR to explicitly disable the `UNIT_ign_TEST` and `UNIT_ModelCommandAPI_TEST` test when a multiple-config CMake generator is used. Note that this test can be run on Windows using any of the available single-config CMake generators, such as Ninja or NMake Makefiles.

Reply:
By the way, while working on this I also got `UNIT_ign_TEST` to work on Windows.